setup.py

```diff
@@ -645,6 +645,10 @@ def run(self):
             report("-- Building NCCL library")
         else:
             report("-- Not using NCCL")
+        if cmake_cache_vars["USE_XCCL"]:
+            report("-- Building XCCL library")
+        else:
+            report("-- Not using XCCL")
```

Comment:
This list keeps growing, I'm not sure if anyone is reading it. Can you elaborate why this is needed other than to mimic behavior similar to NCCL?

Reply:
The results of this list will appear in the final part of the successful build output, but the information may repeat some of the flags printed at the beginning of the build. I think its purpose is to provide some flag information in the log after an incremental build, since small changes to the code do not trigger the message printing from the CMakeLists.txt.